packages/fluent-ui/src/CheckboxesWidget/CheckboxesWidget.tsx

```diff
@@ -17,16 +17,16 @@ const styles_red = {
   fontFamily: `"Segoe UI", "Segoe UI Web (West European)", "Segoe UI", -apple-system, BlinkMacSystemFont, Roboto, "Helvetica Neue", sans-serif;`,
 };
 
-const selectValue = (value: any, selected: any, all: any) => {
+const selectValue = (value: any, selected: any[], all: any) => {
   const at = all.indexOf(value);
   const updated = selected.slice(0, at).concat(value, selected.slice(at));
 
   // As inserting values at predefined index positions doesn't work with empty
   // arrays, we need to reorder the updated selection to match the initial order
-  return updated.sort((a: any, b: any) => all.indexOf(a) > all.indexOf(b));
+  return updated.sort((a, b) => Number(all.indexOf(a) > all.indexOf(b)));
 };
 
-const deselectValue = (value: any, selected: any) => {
+const deselectValue = (value: any, selected: any[]) => {
   return selected.filter((v: any) => v !== value);
 };
```

Comment:
Used in multiple packages, consider moving to utils

Reply:
Will do!